app/models/hosting_environment.rb

```diff
@@ -0,0 +1,13 @@
+module HostingEnvironment
```

Comment:
Hmm, would this be better suited to go in `lib` or `services`? As is, it doesn't seem to model any data, but functions as a service.

Reply:
I'll put it in `lib`